packages/flame/test/components/position_type_test.dart

```diff
@@ -40,6 +40,7 @@ void main() {
     testWithFlameGame(
       'PositionType.viewport',
       (game) async {
+        game.removeAll([game.world, game.cameraComponent]);
```

Comment:
why?

Reply:
Because otherwise the transform operations that will be made on the canvas won't match the result and since it will make some additional operations for those components.